SRC/maps/doors.asm

```diff
@@ -2878,9 +2879,21 @@ door1EF:
 door1F0:
     LOAD_SPR gfx_metGamma
     END_DOOR
-
+;;;;m2maps: 2 added door sets, copies of earlier doors to make map transitions work
```

Comment:
Why add these new doors instead of modifying the existing ones?

Reply:
This was strictly a matter of me not knowing yet if LAMP or ROSE need those pointers at specific ROM locations due to how they handle the data. I'll check with Conner and Liam and if that's not the case, I will happily modify the originals.

Reply:
OK, that's what I suspected, all good